resources/measures/adjust_occupancy_schedule/resources/os_lib_schedules.rb

```diff
@@ -0,0 +1,933 @@
+# *******************************************************************************
```

Comment:
Are the method used from this part of openstudio-standards yet? If so, use from there and remove resource file if possible.

Reply:
I was [tracking](https://nrel.sharepoint.com/:w:/r/sites/comstock/Shared%20Documents/ComStock%20Filing%20Cabinet/ComStock%20Core/openstudio-standards%20refactor%20changes/openstudio-standards%20refactor%20change%20list.docx?d=w94e52e177b154a719fc3cae4e51fd6b4&csf=1&web=1&e=toT71v) this as a revision to make once the whole standards refactor is complete, but I think the methods used have already been implemented and could possibly be updated now.

Reply:
I agree with @asparke on this.
There may be an issue updating from this resource file to the new schedules methods.
I suggest copying these openstudio-standards schedule methods directly into the top of the measure.rb. It'll make it much easier to change them out in the refactor, and we won't have any surprises in them behaving differently:
```
- schedule_get_min_max
- schedule_ruleset_simple_value_adjust
```